posts/lets_talk_about_9_to_5.md

```diff
@@ -0,0 +1,19 @@
+# Let's talk about 9 to 5
+
+We have a flexible policy towards software engineering work hours. Preferring our engineers to be responsible and ensure that their commitments are met rather than measuring work hours has always been our policy. In this blog post I'd like to argue that measuring work hours can indeed be sensible for measuring a teams effectiveness.
+
+For me it all comes back to the idea that symptoms and illnesses are inextricably linked but while you can detect illnesses via their symptoms, you cure them by addressing the underlying sickness.
```

Comment:
I 100% understand why the metaphor is interesting. But maybe it is a bit too dark? Is there an equivalent that doesn't associate work hours and "plague"? :)